src/runtime/methodbinder.cs

```diff
@@ -926,7 +926,9 @@ internal virtual IntPtr Invoke(IntPtr inst, IntPtr args, IntPtr kw, MethodBase i
                 }
 
                 value.Append(": ");
+                Runtime.PyErr_Fetch(out var errType, out var errVal, out var errTrace);
```

Comment:
This might also be a fix for #1371

Reply:
It might let us reenable the test case, but the issue that caused it will still be there.